packages/blocks/src/page-block/default/components.ts

```diff
@@ -95,9 +100,10 @@ export function SelectedRectsContainer(rects: DOMRect[]) {
     <div class="affine-page-selected-rects-container">
       ${rects.map(rect => {
         const style = {
+          position: 'absolute',
           display: 'block',
-          left: rect.left + 'px',
-          top: rect.top + 'px',
+          left: BLOCK_CHILDREN_CONTAINER_PADDING_LEFT + 'px',
```

Comment:
incorrect padding in nested list

![image](https://user-images.githubusercontent.com/14026360/216776962-70622a81-b08f-41f1-8088-5cf35088da7d.png)